psydac/fem/projectors.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+from psydac.linalg.kron     import KroneckerDenseMatrix
+from psydac.core.bsplines   import hrefinement_matrix
+from psydac.linalg.identity import IdentityStencilMatrix
+from psydac.linalg.stencil  import StencilVectorSpace
+
+def knots_to_insert(coarse_grid, fine_grid, tol=1e-14):
```

Comment:
Please provide a docstring.

Reply:
done in [86f53a3](https://github.com/pyccel/psydac/pull/241/commits/86f53a3e46503da7dac62c58b2edeedc4f631418)